modules/oe_theme_helper/src/TwigExtension/TwigExtension.php

```diff
@@ -490,4 +510,72 @@ public function toEclIcon(array $context, $icon): array {
     ];
   }
 
+  /**
+   * Trim given input using smart_trim module heuristics.
+   *
+   * @param \Twig_Environment $env
+   *   Current Twig environment.
+   * @param mixed $input
+   *   Input to be trimmed, it can be a string, an object or a render array.
+   * @param int $limit
+   *   Amount of text to allow.
+   *
+   * @return mixed
+   *   The trimmed output.
+   */
+  public function smartTrim(\Twig_Environment $env, $input, $limit) {
```

Comment:
Can we put the type hinting on the limit?

Reply:
I'm a bit hesitant on this as it might come in a form of a template variable as a string.

Reply:
Yep indeed, let's leave it like this.